client/modules/User/pages/SignupView.jsx

```diff
@@ -59,10 +59,26 @@ class SignupView extends React.Component {
   }
 }
 
+function asyncErrorsSelector(formName, state) {
```

Comment:
is this is code necessary for keeping around around multiple asyncErrors? if yes, that's cool, i'm just curious if there's a way to do this that feels slightly less hacky, that's built into redux form.